Civi/Api4/Query/Api4SelectQuery.php

```diff
@@ -524,7 +537,9 @@ public function getField($expr, $strict = FALSE) {
     if ($strict && !$field) {
       throw new \API_Exception("Invalid field '$fieldName'");
     }
-    $this->apiFieldSpec[$expr] = $field;
+    else {
```

Comment:
why add else here - it's already implicit by the exception throwing above

Reply:
Whoops. I added this to avoid adding `NULL` values to `$apiFieldSpec`, but I used the wrong condition. Should be `if ($field)` instead of `else`. Fixed now.